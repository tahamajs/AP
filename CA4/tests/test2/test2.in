report_team_salary 1
report_team_salary 2
report_team_salary 3
update_team_bonus 1 10
update_team_bonus 5 10
update_team_bonus 2 120
report_team_salary 1
