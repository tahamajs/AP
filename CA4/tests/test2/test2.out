ID: 1
Head ID: 2
Head Name: GolnazEmadi
Team Total Working Hours: 443
Average Member Working Hours: 221.5
Bonus: 0
---
Member ID: 2
Total Earning: 53761
---
Member ID: 4
Total Earning: 46749
---
ID: 2
Head ID: 3
Head Name: ParvinKarami
Team Total Working Hours: 428
Average Member Working Hours: 214.0
Bonus: 0
---
Member ID: 1
Total Earning: 21250
---
Member ID: 3
Total Earning: 54857
---
TEAM_NOT_FOUND
OK
TEAM_NOT_FOUND
INVALID_ARGUMENTS
ID: 1
Head ID: 2
Head Name: GolnazEmadi
Team Total Working Hours: 443
Average Member Working Hours: 221.5
Bonus: 10
---
Member ID: 2
Total Earning: 59137
---
Member ID: 4
Total Earning: 51424
---
